### PR TITLE
add prefix to filepath 

### PR DESCRIPTION
**Pull request by aasheeshkumar**

### CODE REVIEW

File: aliyun.go [hunk 1]
```diff
@@ -62,9 +62,9 @@ func (d *AliyunDownstream) Get(OssCachePath string, DownloadModelFilePath string
 
 // Get file from oss
 func (d *AliyunDownstream) GetObject(OssFileName string) ([]byte, error) {
-
+	cachePath := filepath.Join(d.prefix, OssFileName)
 	// Download the object into ReadCloser(). The body needs to be closed
-	body, err := d.b.GetObject(OssFileName)
+	body, err := d.b.GetObject(cachePath)
```
Comment (bmishra) [inline aliyun.go]: ```body, err := d.b.GetObject(filepath.Join(d.prefix, OssFileName))```

Comment (bmishra) [reply aliyun.go]: no need to add new var

Comment (aasheeshkumar) [reply aliyun.go]: just following coding structure used in other functions , we are creating variable

Comment (bmishra) [reply aliyun.go]: okay, you may introduce new var when it's going to be reused again, or when it is not obvious in-place, or when it helps make the code statement more readable.

**Pull request merged**